e2e/ldap.localhost.json

```diff
@@ -0,0 +1,4 @@
+{
+    "ldapServer": "localhost",
+    "ldapPort": 389
+}
```

Comment:
IMO, this adds extra step in local dev.  I suggest to move it directly into the `env` of cypress.json.